src/menu/menu.c

```diff
@@ -1030,6 +1030,13 @@ init_rootmenu(struct server *server)
 	if (!menu) {
 		current_menu = NULL;
 		menu = menu_create(server, "root-menu", "");
+
+		current_item = item_create(menu, _("Terminal"), false);
+		fill_item("name.action", "Execute");
+		fill_item("command.action", "lab-sensible-terminal");
```

Comment:
Should we also put that into the example `menu.xml` for the root menu? I am not sure on the right approach, with `lab-sensible-terminal` no expected to be available it might be fine I guess. We could then also use the chance to sync it with our actual hardcoded defaults like we do with `rc.xml.all`.

Reply:
Have updated menu.xml